server/services/return-of-service.ts

```diff
@@ -4,7 +4,28 @@ import { rosError } from '../constants';
 import { getParticipantHiredStatuses } from './participant-status';
 import logger from '../logger';
 
-const getRosParticipantStatus = async (participantId) => {
+interface RosParticipant {
```

Comment:
Hmm my concern with at least this naming is that RosParticipant might consist of more than these properties, even if this is all we need for validation. Will be checking on this